public/index.css

```diff
@@ -4,27 +4,29 @@
 }
 
 :root {
-  --user-font-scale: 1rem - 16px;
-  --max-width: calc(100% - 2rem);
+  /* --user-font-scale: 1rem - 16px; */
+  --user-font-scale: calc(1em+1vw);
+  /* --max-width: calc(100% - 2rem); */
+  --max-width:100%;
 }
 
-@media (min-width: 50em) {
+/* @media (min-width: 50em) {
   :root {
     --max-width: 46em;
   }
-}
+} */
```

Comment:
Should this not be completely removed?

Reply:
aye man, however im not sure about the github etiquette for such things. So I was swithering if I should remove someone else's code or just comment it out, I thought I would go for the later, if it was approved then I would look to remove them,

Reply:
Oh not too sure, but since Git will keep history I always tend to just remove obsolete code right away.